setup.py

```diff
@@ -96,10 +96,21 @@ def parse_readme():
                 "data/Dockerfiles/dev/*",
                 "data/Dockerfiles/hardhat/*",
                 "data/Dockerfiles/tendermint/*",
+                "data/contracts/*",
+                "data/contracts/component_registry/*",
+                "data/contracts/component_registry/build/*",
+                "data/contracts/registries_manager/*",
+                "data/contracts/registries_manager/build/*",
+                "data/contracts/service_registry/*",
+                "data/contracts/service_registry/build/*",
                 "test_tools/data/*",
-            ]
+            ],
         },
-        packages=find_packages(include=["autonomy*"]),
+        packages=find_packages(include=["autonomy*"])
+        + [
+            "packages.valory.contracts.component_registry",
+            "packages.valory.contracts.registries_manager",
```

Comment:
why only some of them?

Reply:
It was a mistake, none of them were supposed to be there